### ISSUE TITLE
Upgrade neighbourhoods in San Luis Obispo, CA

### ISSUE THREAD

**Issue by stepps00** [whosonfirst-data/whosonfirst-data#433]
### tl;dr

Status:
- [] Propose **sources**
- [] Establish **concordances** between WOF and source
- [ ] **Upgrade / add** WOF shapes
- [ ] **Add new or missing** features to WOF
- [ ] **Deprecate / supersede** WOF features
- [ ] **Draft review**
- [ ] **Draft revisions** based on review comments
- [ ] **Final review**
- [ ] **Final revisions** based on review comments
- [ ] **Import** into Who's On First

### Details

Sourcing:
- Permissive license?
- Add new src to Who's On First

Upgrade / add:
- `macrohood`
- `neighbourhood`
- `microhood`

Add new or missing features to WOF
- `macrohood`
- `neighbourhood`
- `microhood`

Deprecate / supersede:
- `macrohood`
- `neighbourhood`
- `microhood`

**Comment by stepps00** [comment]
Per our conversation with @oeon, adding a separate issue to track neighbourhood updates in San Luis Obispo, CA.

**Suggested source for SLO:** [SLO City Open Data](http://geographicdata.slocity.opendata.arcgis.com/datasets/367c48b7be614418b0b2e772e83b2928_0)
